packages/@webex/webex-core/src/interceptors/redirect.js

```
@@ -97,6 +98,34 @@ export default class RedirectInterceptor extends Interceptor {
+      return this.webex.request(options);
+    } else if (
+      response.headers &&
+      response.body &&
+      response.body.code === APPAPI_REDIRECT_ERROR &&
+      response.body.data &&
+      response.body.data.siteFullUrl
+    ) {
+      options = clone(options);
+
+      this.webex.logger.warn('redirect: url redirects needed from', options.uri);
+      if (response.options && response.options.qs) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)